docker/env_compass.yaml

```diff
@@ -0,0 +1,16 @@
+#env for COMPASS
+
+  - isce3
+  - xarray
+  - progressbar
+  - rasterio
+  - dem_stitcher
+  - gxx_linux-64>=9,<10
+  - pysolid
+  - pytest
+  - pytest-order
+
+
+
+./CSLC_CALVAL_DELIVERY.playground/run_mem_profiler.sh
+./CSLC_CALVAL_DELIVERY.playground/start_docker_container_for_mem_profile.sh
```

Comment:
Should this be changed to Final?

Reply:
This file `env_compass.yaml` was pushed by accident. I'll retract this file. Thanks for pointing this.